### PR TITLE
LPS-127064 sf

### PR DESCRIPTION
**Pull request by brunofariasdeo**
Related Issue: https://issues.liferay.com/browse/LPS-127064

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js [hunk 1]
```diff
@@ -129,6 +139,11 @@ const LocalesDropdown = ({
 									<ClayLayout.ContentCol containerElement="span">
 										<AvailableLocaleLabel
 											isDefault={isDefault}
+											isSubmitLabel={
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js:144]: E se tu fiz só `isSubmitLabel={fieldName === 'submitLabel'}` não funciona não?

**Review by carolmariaabb — COMMENTED** [review]
@brunofariasdeo comentei só duas besteirinhas..

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -267,6 +268,7 @@ no-element-sets-were-found=No element sets were found.
 no-forms-were-found=No forms were found.
 no-value-is-defined-for-field-x=No value is defined for field {0}.
 no-x-were-found=No {0} were found.
+not-customized=Not customized
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties:271]: @brunofariasdeo Se tu não colocar ponto todas as palavras tem que começar com letra maiúscula `Not Customized`, regra de Brian

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:stop

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @brunofariasdeo, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
Resent [here](https://github.com/carolmariaabb/liferay-portal/pull/302).

**Pull request closed without merging**